src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterGood.java

```diff
@@ -36,4 +36,9 @@ public void check5() {
         try {} finally {}
         try {} catch (Exception e){} finally {}
     }
+
+    public void check6() {
+        synchronized (this) { }
+        synchronized (this) { }
```

Comment:
what is the point to duplicate line above?

Reply:
I have removed it